src/utils/hooks/useDeleteTicketMessage.tsx

```diff
@@ -4,12 +4,12 @@ import { useDispatch } from 'react-redux';
 import { deleteTicketMessage } from '../../api/registrationApi';
 import { setNotification } from '../../redux/notificationSlice';
 
-export const useDeleteTicketMessage = (ticketId: string, refetchData?: () => void) => {
+export const useDeleteTicketMessage = (messageId: string, refetchData?: () => void) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
 
   return useMutation({
-    mutationFn: (messageId: string) => deleteTicketMessage(ticketId, messageId),
+    mutationFn: () => deleteTicketMessage(messageId),
```

Comment:
Kunne bevart `messageId` som parameter her også, så kunne man brukt samme hook til å slette flere meldinger i stedet for å måtte initiere en ny hook for hver melding man skal slette. Men om vi aldri har det behovet går det for det samme